### ISSUE TITLE
pretrained model?

### ISSUE THREAD

**Issue by zhangxiaosa** [Hippogriff/CSGNet#1]
Hello, is there any pretrained models?

**Comment by Hippogriff** [comment]
Updated README.md with pre-trained model link. Thanx.

**Comment by adrelino** [comment]
How was the uploaded pre-trained model trained? Considering it's name, it seems like the hyperparameters set in config_synthetic.xml (but with encoder_drop = 0.0 instead of 0.2) and train_synthetic.py was used.

Especially I am interested in the training/val/test split used so I can do an experiment comparable to yours. In train_synthetic, only dataset sizes k=3,5,7 appear, but in test_synthetic, you also have 9,11,13...   Would be grateful for your help trying to reproduce your results or at least be able to compare to them...

**Comment by Hippogriff** [comment]
If it was not clear, dropout was used only in the decoder because the number of parameters in the decoder were really large.
If you want to train on larger sequence of programs use this in the training script too:

```
data_labels_paths = {3: "data/synthetic/one_op/expressions.txt",
                     5: "data/synthetic/two_ops/expressions.txt",
                     7: "data/synthetic/three_ops/expressions.txt",
                     9: "data/synthetic/four_ops/expressions.txt",
                     11: "data/synthetic/five_ops/expressions.txt",
                     13: "data/synthetic/six_ops/expressions.txt"}
# first element of list is num of training examples, and second is number of
# testing examples.
proportion = config.proportion  # proportion is in percentage. vary from [1, 100].
dataset_sizes = {
    3: [30000, 50 * proportion],
    5: [110000, 500 * proportion],
    7: [170000, 500 * proportion],
    9: [270000, 500 * proportion],
    11: [370000, 1000 * proportion],
    13: [370000, 1000 * proportion]
}
```
